src/main/java/uk/nhs/hee/tis/trainee/sync/service/TcsSyncService.java

```diff
@@ -149,6 +146,7 @@ private void syncDetails(TraineeDetailsDto dto, String apiPath, Operation operat
             dto.getTraineeTisId());
         break;
       case DELETE:
+        personService.deleteById(dto.getTraineeTisId());
```

Comment:
I am going to amend the call below yours to check the apiPath that is being used, since we currently only implement the REST DELETE endpoint for Programme Memberships. I think you'd want to possibly restrict your deleteById() call as well - to records of type Person? e.g.
if (apiPath.equals(TABLE_NAME_TO_API_PATH.get(TABLE_PERSON))) {